holidays/RELEASE.rst

```diff
@@ -0,0 +1,69 @@
+How to release a new version of Python Holidays
+===============================================
+
+- Finalize the current development version
+
+  - switch to ``beta`` branch and pull the most recent changes
+    from https://github.com/vacanza/python-holidays remote ``beta`` branch.
+  - generate release notes by running the following script
+    ``scripts/generate_release_notes.py -t <version>``, where <version> is the
+    value of tag/version you're going to release, e.g. 0.39
+  - insert the script's output into the top of ``CHANGES`` file
+    (see previous release notes for consistent formatting)
+  - commit the updated ``CHANGES`` file to ``beta`` branch with the following
+    commit message 'Finalize v<version>', e.g. 'Finalize v0.39'
+  - push changes to https://github.com/vacanza/python-holidays ``beta`` branch
+  - make sure the push related CI/CD job(s) have been completed successfully
+
+- Merge the finalized changes into ``master`` branch:
+
+  - create a new PR for the recent changes from ``beta`` to ``master`` branch
+    using 'v<version>' as a PR title and the previously generated release notes
+    as a PR description
+  - get the PR reviewed by at least one of the code owners
+  - merge the PR into ``master`` branch by enabling auto-merge with
```

Comment:
A bit unclear about "enabling auto-merge" at this stage.

Reply:
👍 
Removed that.